packages/foreman/rubygem-hammer_cli_foreman/rubygem-hammer_cli_foreman.spec

```diff
@@ -25,7 +23,7 @@ Source0: https://rubygems.org/gems/%{gem_name}-%{version}%{?prerelease}.gem
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(hammer_cli) >= 2.1.0
+Requires: %{?scl_prefix}rubygem(hammer_cli) >= 3.7.0
```

Comment:
Note to self: update this gem spec in develop. This should be autogenerated nowadays.